src/Symfony/Bundle/FrameworkBundle/Resources/config/collectors.xml

```diff
@@ -17,17 +17,17 @@
 
     <services>
         <service id="data_collector.config" class="%data_collector.config.class%" public="false">
-            <tag name="data_collector" template="@WebProfiler/Collector/config.html.twig" id="config" priority="255" />
+            <tag name="data_collector" template="@WebProfiler/Collector/config.html.twig" id="config" priority="-255" />
             <call method="setKernel"><argument type="service" id="kernel" on-invalid="ignore" /></call>
         </service>
 
         <service id="data_collector.request" class="%data_collector.request.class%">
             <tag name="kernel.event_subscriber" />
-            <tag name="data_collector" template="@WebProfiler/Collector/request.html.twig" id="request" priority="255" />
+            <tag name="data_collector" template="@WebProfiler/Collector/request.html.twig" id="request" priority="260" />
```

Comment:
@javiereguiluz I would even suggest leaving holes between the priorities used in Symfony to make it easier to inject things between them